Misc/NEWS.d/next/Library/2022-01-20-10-35-10.bpo-46434.geS-aP.rst

```diff
@@ -0,0 +1,2 @@
+pdb help now gracefully handles any case where __doc__ is missing, for
+example when run from a windows embeddable package
```

Comment:
```suggestion
:mod:`pdb` now gracefully handles any case where :attr:`__doc__` is missing, for
example when run with pregenerated optimized ``.pyc`` files.
```

FYI, I found the special references by searching the `Doc` folder for other examples. These will turn them into links in the final doc.

Reply:
Ah cool thank you, good to know! Maybe it's not important but should we retain a reference to `help` in there given it's the help command within pdb which has been fixed? Otherwise "any case" without the scope of 'help' feels a big of an overstatement (!)

Reply:
Hopefully ba2053d34c is ok